node/src/accountant/db_access_objects/payable_dao.rs

```diff
@@ -61,7 +88,10 @@ pub trait PayableDao: Debug + Send {
         confirmed_payables: &[PendingPayableFingerprint],
     ) -> Result<(), PayableDaoError>;
 
-    fn non_pending_payables(&self) -> Vec<PayableAccount>;
+    fn non_pending_payables(
```

Comment:
This should probably stop mentioning "non_pending_" here. It should be just `return_records` or so.